action/sealedenvelope.go

```diff
@@ -24,6 +24,19 @@ type SealedEnvelope struct {
 	signature    []byte
 }
 
+// NewSealedEnvelope creates a new SealedEnvelope
+func NewSealedEnvelope(elp Envelope, encoding iotextypes.Encoding,
```

Comment:
probably not the way we want to go about dealing `SealedEnvelope`

i think for this PR, should wrap web3 staking into the special-addressed execution and send it. See comments below. If we want to change to `sendRaw(selp SealedEnvelope)`, let's have another PR

Reply:
after discusion,  ` NewSealedEnvelope` is removed